Source/WebCore/css/htmlSwitchControl.css

```diff
@@ -27,3 +27,7 @@
 input[type="checkbox"][switch] {
     margin: initial;
 }
+
+input[type="checkbox"][switch]::thumb, input[type="checkbox"][switch]::track {
+    appearance: inherit !important;
+}
```

Comment:
Would have been nice to have tests for this

Reply:
We can add ref tests once painting is implemented.